packs/data/feat-effects.db/effect-elemental-assault.json

```diff
@@ -0,0 +1,70 @@
+{
+    "_id": "XaZdQHF9GvaJINqH",
+    "img": "systems/pf2e/icons/spells/elemental-zone.webp",
+    "name": "Effect: Elemental Assault",
+    "system": {
+        "badge": null,
+        "description": {
+            "value": "<p>You shroud your arms and held weapons in elemental magic. Choose one element. Until the end of your next turn, your Strikes deal an additional [[/r 1d6]] damage of the indicated type and have the trait corresponding to the element: electricity for air, bludgeoning for earth, fire for fire, or cold for water.</p>"
+        },
+        "duration": {
+            "expiry": "turn-end",
+            "sustained": false,
+            "unit": "rounds",
+            "value": 1
+        },
+        "level": {
+            "value": 1
+        },
+        "rules": [
+            {
+                "choices": [
+                    {
+                        "label": "PF2E.TraitAir",
+                        "value": "electricity"
+                    },
+                    {
+                        "label": "PF2E.TraitEarth",
+                        "value": "bludgeoning"
+                    },
+                    {
+                        "label": "PF2E.TraitFire",
+                        "value": "fire"
+                    },
+                    {
+                        "label": "PF2E.TraitWater",
+                        "value": "cold"
+                    }
+                ],
+                "flag": "elementalAssault",
+                "key": "ChoiceSet",
+                "prompt": "PF2E.SpecificRule.Suli.ElementalAssult.Prompt",
+                "rollOption": "elementalAssault"
```

Comment:
We don't need to set a roll option do we? If we do the roll option should be kebab case.